### PR TITLE
Restyle pricing form and create template for form with quote

### PR DESCRIPTION
**Pull request by zchase**
Add some styling to the Contact Us form on the pricing page. If we agree it looks good, I will go ahead and apply this style to the remaining forms.

Old Form:
oldForm

New Form:
formux

### CODE REVIEW

File: assets/sass/_hubspot.scss [hunk 1]
```diff
@@ -91,7 +91,16 @@
 
     &.pricing {
         label {
-            @apply font-normal;
+            @apply font-normal text-base;
+        }
+        input, textarea {
+            @apply shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight text-xl;
+            &:focus {
+                @apply outline-none shadow-outline;
```
Comment (cnunciato) [inline assets/sass/_hubspot.scss]: I'm not sure I understand the intent with the borders and outlines. Is there some browser inconsistency you're solving for?

File: assets/sass/_hubspot.scss [hunk 2]
```diff
@@ -91,7 +91,16 @@
 
     &.pricing {
         label {
-            @apply font-normal;
+            @apply font-normal text-base;
+        }
+        input, textarea {
+            @apply shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight text-xl;
```
Comment (cnunciato) [inline assets/sass/_hubspot.scss]: Let's remove the `shadow` around the form fields and keep them on the same plane as the form itself.

File: layouts/partials/form-section.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                <h3 class="text-center">{{ .formHeader }}</h3>
+                <div class="mx-auto flex justify-center">
+                    <div class="hs-form hs-form-fg-dark font-normal">
+                        {{ partial "hubspot-form.html" (dict "hubspotFormID" .hubspotFormID "classNames" .formClassNames) }}
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
```
Comment (cnunciato) [inline layouts/partials/form-section.html]: Newline. 😄 (If you're using VSCode, you can set this with `"files.insertFinalNewline": true`.)

File: assets/sass/_hubspot.scss (hunk 1 above)
Comment (zchase) [reply assets/sass/_hubspot.scss]: Yup I am just making sure the focus outline is consistent across browsers. In my changes I updated the outline color to be purple to fit more in line with out theme. I'm interested to to get your opinion on this.

File: assets/config/tailwind.config.js [hunk 4]
```diff
@@ -86,6 +86,10 @@ module.exports = {
                     ...defaultTheme.fontFamily.sans,
                 ],
             },
+
+            boxShadow: {
+                "input-outline": "0 0 0 3px rgba(81, 38, 104, 0.5)"
```
Comment (cnunciato) [inline assets/config/tailwind.config.js]: I'm up for giving this a try, though I'd use a color from the maps defined above in this page, and probably one of the lighter-to-medium grays, for portability.

File: assets/sass/_hubspot.scss [hunk 5]
```diff
@@ -93,5 +93,17 @@
...
             @apply font-normal;
         }
+        input, textarea {
+            @apply appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight text-xl;
+            &:focus {
+                @apply outline-none shadow-input-outline;
+            }
+        }
+        .hs-button {
+            @apply text-white btn-orange;
```
Comment (cnunciato) [inline assets/sass/_hubspot.scss]: Let's have this button stick with the blue and unaltered font size (as with the submit button below), which should be the defaults: 

image

**Pull request merged**